frontend/src/config/theme.js

```diff
@@ -34,7 +34,10 @@ export const lightTheme = {
   ...lightBaseColors,
   body: "#ffffff",
   text: "#323739",
-
+  plannerCartCardHover: "#f4f4f4",
```

Comment:
Nitpick, include this styling within `plannerCartCard` key. So something like the below would be better

```
plannerCartCard: {
    backgroundColorHover: "#f4f4f4",
}
```
This is because we can add future variables under `plannerCartCart` key later on if we need it.